code/modules/awaymissions/mission_code/ruins/moonoutpost19.dm

```diff
@@ -0,0 +1,37 @@
+//papers for moonoutpost19.dmm
+
+/obj/item/paper/fluff/ruins/moonoutpost19/lobby
+	name = "a note to Joe"
+	info = "<br>Hey Joe, since you keep forgetting here's the list of lockdown areas and the locations of control buttons. Thank me later.<br><br><b>(Lobby)</b><br><br>- Research Division lockdown<br>- Medical Ward lockdown<br>- Cargo Bay gates<br><br><b>(Cargo Bay)</b><br><br>- Exterior hangar gate<br>- Cargo bay gates<br><br><b>(Senior Researcher's Office)</b><br><br>- Specimen Containtment Zone lockdown<br><br>Note: keep in mind, in case of power outages these buttons won't be functional. In this case you'll find a few generators in engineering wing, set them up properly then report to technicians ASAP."
```

Comment:
```suggestion
	info = "<br>Hey Joe, since you keep forgetting here's the list of lockdown areas and the locations of control buttons. Thank me later.<br><br><b>(Lobby)</b><br><br>- Research Division lockdown<br>- Medical Ward lockdown<br>- Cargo Bay gates<br><br><b>(Cargo Bay)</b><br><br>- Exterior hangar gate<br>- Cargo bay gates<br><br><b>(Senior Researcher's Office)</b><br><br>- Specimen Containment Zone lockdown<br><br>Note: keep in mind, in case of power outages these buttons won't be functional. In this case you'll find a few generators in engineering wing, set them up properly then report to technicians ASAP."
```